.gitlab/auto-deploy-values.yaml

```diff
@@ -2,7 +2,7 @@ ingress:
   annotations:
     nginx.ingress.kubernetes.io/server-snippet: |
       location /sitemaps/sitemap.xml.gz {
-        return 302 https://navodydigital.s3.eu-central-1.amazonaws.com/sitemap.xml.gz;
+        return 302 https://navodydigital.s3.eu-central-1.amazonaws.com/sitemaps/sitemap.xml.gz;
```

Comment:
@celuchmarek potom ked toto mame moze ist teoreticky do ENV `SITEMAP_PUBLIC_URL=https://navody.digital/sitemaps/sitemap.xml.gz` nech to nie je roztrusene

Reply:
jo, to som upravil tiež

Reply:
nooo, v https://navody.digital/robots.txt je stale S3 URLka

Reply:
aha, sorry. Ja som si všímal tú časť "sitemaps", nie doménu. Hej to dáva zmysel. Ok, to tam dám aj s posledným fixom.